todo-app/src/components/Todos/List-todos.js

```diff
@@ -0,0 +1,69 @@
+// jshint esnext: true
+
+import React from 'react';
+import TodoItem from './Todo-item';
```

Comment:
It would be a little nicer to name your component files exactly as the component is named, so that it's immediately obvious what it is. So something like:

    import TodoItem from './TodoItem';

Reply:
The same thing with what to use camel case or something else, I think I'll just do camel case from now on it's usable in the js code as well

Reply:
What do you mean? React components will always need an uppercase first letter. Besides React components, yeah, camel case is a safe bet for all JS.